### PR TITLE
Fix CS sniffer sync.

### PR DESCRIPTION
**Pull request by dereuromark**
Refs https://github.com/cakephp/phinx/pull/1330 issues around CS

At least now we know why..
tons of "Missing parameter comment" still in there.
tbh I never really liked this strict one as it only lead to a lot of duplicating without much value.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ before_script:
 script:
     - if [[ $DEFAULT = 1 ]]; then vendor/bin/phpunit --coverage-text --coverage-clover=coverage.clover; fi
     - if [[ $PHPSTAN = 1 ]]; then composer require --dev phpstan/phpstan:^0.8 && vendor/bin/phpstan analyse -c phpstan.neon -l 4 src; fi
-    - if [[ $PHPCS = 1 ]]; then composer require --dev squizlabs/php_codesniffer:~3.1.1 && vendor/bin/phpcs -p -s src; fi
+    - if [[ $PHPCS = 1 ]]; then phpcs -p --standard=vendor/cakephp/cakephp-codesniffer/CakePHP src/ tests/ app/; fi
```
Comment (ceeram) [inline .travis.yml]: cant do `then phpcs -p `
should be `then vendor/bin/phpcs -p `

Comment (ceeram) [reply .travis.yml]: also probs will complain about cakephp standard not being available

Comment (dereuromark) [reply .travis.yml]: Good point, I re-use the  composer command then.

**Comment by JayPHP** [comment]
What's the status of this PR?

**Comment by dereuromark** [comment]
Check the travis report. Too many issues regarding comments in doc blocks.
I suggest just silencing them as they also bear no real value here right now moving forward.

**Comment by lorenzo** [comment]
Closing as the code has diverged significantly

**Pull request closed without merging**